### PR TITLE
Automate the creation of a release after tag push

### PR DESCRIPTION
**Pull request by skyerus**

### CODE REVIEW

File: .goreleaser.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# This is an example .goreleaser.yml file with some sensible defaults.
+# Make sure to check the documentation at https://goreleaser.com
```
Comment (toddbaert) [inline .goreleaser.yaml:2]: Can you remove this comment?

File: .goreleaser.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  - replacements:
+      darwin: Darwin
+      linux: Linux
+      windows: Windows
+      386: i386
+      amd64: x86_64
+checksum:
+  name_template: 'checksums.txt'
+snapshot:
+  name_template: "{{ incpatch .Version }}-next"
```
Comment (toddbaert) [inline .goreleaser.yaml:22]: Does this do anything in our case, if we're only running this on `v*` tags?

**Review by toddbaert — APPROVED** [review]

File: .goreleaser.yaml (hunk 2 above)
Comment (skyerus) [reply .goreleaser.yaml]: Removed

File: .goreleaser.yaml (hunk 1 above)
Comment (skyerus) [reply .goreleaser.yaml]: Done

**Pull request merged**